test/unit/media-play-button.spec.js

```diff
@@ -4,7 +4,7 @@ import { MediaUIAttributes, MediaUIEvents } from '../../src/js/constants';
 import { verbs } from '../../src/js/labels/labels';
 import '../../src/js/media-play-button';
 
-describe('<media-play-button/>', () => {
+describe('<media-play-button>', () => {
```

Comment:
I see you have a preference for our `<component-name>` 😄. Happy to conform unless other folks lean another way.

Reply:
yes we mentioned this before I think, @gkatsev as well. 
custom elements don't have a self closing feature so it could confuse devs. for React components this is fine

Reply:
works for me. I don't have strong opinions here either way.